cmd/oceanbench/api.go

```diff
@@ -91,6 +91,40 @@ func apiHandler(w http.ResponseWriter, r *http.Request) {
 			fmt.Fprint(w, string(enc))
 			return
 
+		case "devices":
+			switch val {
```

Comment:
Should we use a switch for a single case ? Should we just do
```
if val != "site" {
    break
}
```

and then we can reduce the amount of nested code.

Reply:
I've done it like this with the intention that the api will grow over time and there will be more endpoints which can just be added as cases to the existing switch statement without having to adjust the current format. This also keeps it more inline with other endpoints.

Reply:
IMO we should just change the structure when that happens. I don't think code needs to be extendable to the point that you choose a ```switch``` over ```if``` :laughing: .

Reply:
I don't think the level of nesting is that bad, it might actually help follow the API endpoints better. I would probably do the same as David to save changing it if it's extended in the future aha